openshift/4.0/pipelines/scripts/notify-teams.groovy

```diff
@@ -0,0 +1,71 @@
+import groovy.json.JsonOutput
+import java.util.regex.Pattern
+
+def version = "1.0"
+
+// ------------------
+// Notifications
+// ------------------
+
+def success(appName, version, environment, webhook_url = "") {
+  sendNotification(
+    webhook_url,
+    "${appName}, release: [${version}] was deployed to **${environment}** 🚀",
+    "#1ee321",  // green
+    "https://i.imgur.com/zHzPWXr.png"
+  )
+}
+
+def failure(appName, version, environment, webhook_url = "") {
+  sendNotification(
+    webhook_url,
+    "Could not deploy ${appName}, release: [${version}] to **${environment}** 🤕",
+    "#e3211e",  // red
+    "https://i.imgflip.com/1czxka.jpg"
+  )
+}
+
+def createPayload(text) {
+  def jenkinsAvatar = "https://wiki.jenkins.io/download/attachments/2916393/headshot.png"
+
+  def message = text
+  def payload = JsonOutput.toJson([
+    type: "message",
+    attachments: [
+      [
+        contentType: "application/vnd.microsoft.card.adaptive",
+        contentUrl: null,
+        content: [
+          '$schema': "http://adaptivecards.io/schemas/adaptive-card.json",
+          type: "AdaptiveCard",
+          version: "1.2",
+          body: [
+            [
+              type: "TextBlock",
+              text: message
+            ]
+          ]
+        ]
+      ]
+    ]
+  ])
```

Comment:
This is the basic format for an MS Teams notification "card". We can get fancier with this but for now it just works. 
A future enhancement could update this to include images and links back to the build that was just completed in Jenkins